style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: orangered;
+}
```

Comment:
- [ ] Great job here 👌💯 but according to the requirements, you should use a class to select the `<h1>` tag, So Kindly add a class name to this element `<h1 class="header">` and use it in your CSS file 
`.header {color: orangered;}`👌
- 

![image](https://user-images.githubusercontent.com/45231276/185032956-ca65df88-2c93-4fdc-97c0-93e7050d2989.png)

Reply:
hi there @Bondok6 
can you please check that and confirm if i have completed all required tasks successfully so that i apply for request review again officially.

Reply:
Hi @Zuhaib042,

Great job implementing the requirements ✅, In my opinion, everything is good 👌 
Please, Request another review and a code reviewer will review your code again 💯👌